src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,7 @@
-// Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo';
+
+export const TodoList = ({ todos }) => (
```

Comment:
You are not checking if the 'todos' prop is an array, and if it is not empty. If it is not provided or empty, your component will fail.